flexiblepricing/views/v0/__init__.py

```diff
@@ -63,6 +62,30 @@ def get_queryset(self):
         if status_search is not None:
             queryset = queryset.filter(status=status_search)
 
+        courseware_search = self.request.query_params.get("courseware")
+        if courseware_search is not None:
+            courseware_content_type, courseware_object_id = courseware_search.split(":")
+            course_type = models.ContentType.objects.get(
+                app_label="courses", model="course"
+            )
+            program_type = models.ContentType.objects.get(
+                app_label="courses", model="program"
+            )
+            if courseware_content_type == "course":
+                queryset = queryset.filter(
+                    Q(
+                        courseware_object_id=courseware_object_id,
+                        courseware_content_type=course_type,
+                    )
+                )
+            elif courseware_content_type == "program":
+                queryset = queryset.filter(
+                    Q(
+                        courseware_object_id=courseware_object_id,
+                        courseware_content_type=program_type,
+                    )
+                )
+
```

Comment:
We can simplify this to:
```suggestion
            content_type = ContentType.objects.get(
                app_label="courses", model=courseware_content_type
            )
            queryset = queryset.filter(
                Q(
                    courseware_object_id=courseware_object_id,
                    courseware_content_type=content_type,
                )
            )

```